services/workers/src/main/java/org/sagebionetworks/table/worker/TableQueryNextPageWorker.java

```diff
@@ -53,6 +56,8 @@ public void run(ProgressCallback progressCallback, Message message)
 			// This means we cannot use this table
 			asynchJobStatusManager.setJobFailed(status.getJobId(), e);
 		}catch(Throwable e){
+			// Attempt to translate the exception into a 'user-friendly' message.
+			RuntimeException translatedException = tableExceptionTranslator.translateException(e);
```

Comment:
This translatedException is not used, you might need to set this in the job status: asynchJobStatusManager.setJobFailed(status.getJobId(), translatedException);